frontend/src/App.js

```diff
@@ -1,9 +1,40 @@
 import React from 'react'
+import { BrowserRouter, Route, Switch } from 'react-router-dom'
+import { Provider } from 'react-redux'
+import { configureStore, combineReducers } from '@reduxjs/toolkit'
+
+import Header from './components/Header'
+import Home from './Pages/Home'
+import SignUp from './Pages/SignUp'
+import Profile from './Pages/Profile'
+
+import user from './reducers/user'
+
+const reducer = combineReducers({
+  user: user.reducer
+})
+
+const store = configureStore({ reducer })
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <BrowserRouter>
+      <Provider store={store}>  
+        <Header />
+        <main>
+          <Switch>
+            <Route exact path='/'>
+              <Home />
+            </Route>
+            <Route exact path='/signup'>
```

Comment:
When we update the page in Signup page we are receiving a 404 error from netlify, see picture 
![image](https://user-images.githubusercontent.com/74493943/120532403-ef02e600-c3df-11eb-9cfb-ad1f8bedc354.png)

Reply:
To fix it you need to create `_redirects` file in the `public` directory of your react project and paste it inside:
``` 
/* /index.html 200
```